schemas/context/opportunity-crm.schema.json

```diff
@@ -0,0 +1,158 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/opportunity-crm",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "CRM Opportunity",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/xdm/context/opportunity"
+  ],
+  "description": "CRM specific opportunity fields.",
+  "definitions": {
+    "opportunity-crm": {
+      "properties": {
+        "xdm:opportunityName": {
+          "title": "Opportunity Name",
+          "description": "Subject or descriptive name, such as the expected order or company name, for the opportunity.",
+          "type": "string"
+        },
+        "xdm:sourceType": {
+          "title": "Opportunity Source Type",
+          "description": "Opportunity source type",
+          "type": "string"
+        },
+        "xdm:description": {
```

Comment:
Should we rename this as "opprtunityDescription" as we have description in other schemas as well ?